plugins/de.cau.cs.kieler.kgraph.language.server/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/,\
+           xtend-gen/
+bin.includes = .,\
+               META-INF/,\
+               plugin.xml,\
+               about.html
+bin.excludes = **/*.xtend
+src.includes = about.html
```

Comment:
The bin.includes differ from the klighd.lsp binary includes, which one is correct?

Reply:
Some bundles seem to not have an about.html or plugin.xml file, so they are also not included in those cases. As I would think, every bundle should have that, so some older bundles are unconsistent in this way yet.